src/renderers/dom/shared/ReactInputSelection.js

```diff
@@ -30,12 +33,16 @@ function isInDocument(node) {
  */
 var ReactInputSelection = {
   hasSelectionCapabilities: function(elem) {
-    var nodeName = elem && elem.nodeName && elem.nodeName.toLowerCase();
+    if (elem === window) {
+      return false;
+    }
```

Comment:
@sebmarkbage this is the only downside I see to calling the `nodeName` getter. Occasionally this value is the window. That's because the target instance falls back to the window when it can't find a target instance:

https://github.com/facebook/react/blob/master/src/renderers/dom/shared/eventPlugins/ChangeEventPlugin.js#L149

I can't help but wonder if this could be `document.body` instead of window.